File: Famoser.MassPass.Data/Entities/Communications/Response/ContentEntityHistoryResponse.cs
using System.Collections.Generic;
using Famoser.MassPass.Data.Entities.Communications.Response.Base;
using Famoser.MassPass.Data.Entities.Communications.Response.Entitites;

namespace Famoser.MassPass.Data.Entities.Communications.Response
{
    public class ContentEntityHistoryResponse : ApiResponse
    {
        public ContentEntityHistoryResponse()
        {
            HistoryEntries = new List<HistoryEntry>();
        }

        public List<HistoryEntry> HistoryEntries { get; set; }
    }
}

File: Famoser.MassPass.Api/src/Models/Response/ContentEntityHistoryResponse.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 23.05.2016
 * Time: 09:49
 */

namespace Famoser\MassPass\Models\Response;


use Famoser\MassPass\Models\Response\Base\ApiResponse;

class ContentEntityHistoryResponse extends ApiResponse
{
    /*
     * @var Entities\HistoryEntry[]
     */
    public $HistoryEntries;
}